bin/fixmystreet.com/add_emergency_message

```diff
@@ -35,42 +40,60 @@ if (!$opts->commit) {
     say colored("*** DRY RUN ***", 'cyan');
 }
 
-my $field = {
-    order => 0,
-    required => 'true',
-    protected => 'true',
-    code => $opts->code,
-    description => $opts->question,
-    datatype => 'singlevaluelist',
-    variable => 'true',
-    values => [
-        {
-            key => 'yes',
-            name => $opts->yes,
-            disable => 1,
-            disable_message => $opts->message,
-        },
-        {
-            key => 'no',
-            name => $opts->no,
-        }
-    ],
-};
+my $field;
+
+if ( $opts->mode eq 'questions' ) {
+    say STDERR "questions, code, yes and no required"
+        unless $opts->questions && $opts->code && $opts->yes && $opts->no;
+
+    $field = {
+        order => 0,
+        required => 'true',
+        protected => 'true',
+        code => $opts->code,
+        description => $opts->question,
+        datatype => 'singlevaluelist',
+        variable => 'true',
+        values => [
+            {
+                key => 'yes',
+                name => $opts->yes,
+                disable => 1,
+                disable_message => $opts->message,
+            },
+            {
+                key => 'no',
+                name => $opts->no,
+            }
+        ],
+    };
+} else {
+    $field = {
+        order => 0,
+        protected => 'true',
+        disable_form => 'true',
+        code => '_fms_disable_',
+        description => $opts->message,
+        variable => 'false',
+    };
+}
 
 my $body = FixMyStreet::DB->resultset("Body")->find({ name => $opts->body });
 unless ($body) {
-    say STDERR "Could not find body " . $opts->body;
+    die "Could not find body " . $opts->body;
```

Comment:
Afraid you changed the wrong one - this one already exits on the next line, it was line 46 I was suggesting needs to die/exit.

Reply:
doh!